easybuild/easyconfigs/p/pytest/pytest-3.8.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'Bundle'
```

Comment:
@SimonPinches according to the first failing travis test, you have to use the `PythonBundle` easyblock for bundles of python packages.